demoapp/package.json

```diff
@@ -19,15 +19,14 @@
     }
   },
   "dependencies": {
-    "cordova-android": "^7.0.0",
-    "cordova-ios": "^5.0.0",
-    "cordova-plugin-appcenter-analytics": "^0.4.1",
-    "cordova-plugin-appcenter-crashes": "^0.4.1",
-    "cordova-plugin-appcenter-push": "^0.4.1",
-    "cordova-plugin-appcenter-shared": "^0.4.1",
+    "cordova-android": "^7.1.4",
```

Comment:
We should consider pinning npm versions in demoapp like the way we do in RN SDK. I know this is not directly related to this PR, we could improve this later.